content/guides/references/configuration.md

```diff
@@ -101,14 +101,14 @@ you should understand well. The default values listed here are meaningful.
+| -------------------- | --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `downloadsFolder`    | `cypress/downloads`   | Path to folder where files downloaded during a test are saved                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `fileServerFolder`   | root project folder   | Path to folder where application files will attempt to be served from                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `fixturesFolder`     | `cypress/fixtures`    | Path to folder containing fixture files (Pass `false` to disable)                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `excludeSpecPattern` | `*.hot-update.js`     | A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses `minimatch` with the options: `{dot: true, matchBase: true}`. We suggest using [https://globster.xyz](https://globster.xyz) to test what files would match. <br><br> The **/node_modules/** pattern is automatically added to both e2e.specExcludePattern and component.specExcludePattern, and does not need to be specified (and can't be overridden). |
```

Comment:
We originally had `excludeSpecPattern` listed here which sort of makes sense. I also have it listed in each row for the test types so we can show the default. 

We probably don't also need it here. Let me know what you think and I'll clean things up accordingly.

Reply:
The excludeSpecPattern option is not valid at the top-level, and may only be defined inside the component and e2e objects. Please remove it from the general config.